Poker/Game.cs

```diff
@@ -14,29 +14,24 @@ class Game
         string Player_1 = "Player 1";
         string Player_2 = "Player 2";
 
-        List<Card> Hand_1 = new List<Card>();
-        List<Card> Hand_2 = new List<Card>();
-
         public Game()
         {
             Players = new List<Player>();
 
             Players.Add(new Player(Player_1));
             Players.Add(new Player(Player_2));
-             
+
             Deck deck = new Deck();
-            
+
         }
 
-        public void DealCards(List<Card> listOfCards, List<Card> hand)
+        public void DealCards()
         {
-            foreach (Player item in Players)
+            foreach (Player player in Players)
             {
-                hand = listOfCards.Take(2).ToList();
-                listOfCards.RemoveRange(0, 2);
+                player.Hand.Add(deck.DrawCardFromDeck());
+                player.Hand.Add(deck.DrawCardFromDeck());
```

Comment:
I could have put this in a forloop but this will do for now. Why put it in a forloop? What if we wanted to reuse this for 5 card poker? Then we could change the amount of cards to pass in much more easily!